src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -32,6 +32,19 @@ public boolean isFinished() {
         return (timer.hasElapsed(Constants.SHOOTER_TIME_AFTER_TRIGGER)) && (activated);
     }
 
+    @Override
+    public void execute() {
+        if (((RobotContainer.isRedAlliance() == true) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) > 180)) || ((RobotContainer.isRedAlliance() == false) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) < 180))) {
+            shooter.setShooterMotorRightSpeed(Constants.SHOOTER_MOTOR_LEFT_SPEED);
```

Comment:
Right and left are no longer accurate - that could change to high and low speeds.

Reply:
Done